src/main/java/org/embulk/filter/column/ColumnFilterPlugin.java

```diff
@@ -1,5 +1,6 @@
 package org.embulk.filter.column;
 
+import com.github.kysnm.jsonpathcompiler.internal.Path;
```

Comment:
It looks pretty wierd that we can access its internal package.
I understand you simply copied from jsonpath library, but could you reconsider of the package name?